arcgis-rest-api/index.d.ts

```diff
@@ -0,0 +1,221 @@
+// Type definitions for arcgis-to-geojson-utils 10.4
```

Comment:
This has an identical project name to the other package. Could it be named `ArcGIS Rest API` instead?